vscode-antimony/src/server/stibium/stibium/symbols.py

```diff
@@ -203,6 +208,30 @@ def help_str(self):
                     "Initialized Value:" + init_val)
         else:
             ret += '\n({}) {}\n'.format(self.type, self.name)
+        if self.events:
+            for event in self.events:
+                if event.get_name_text():

```

Comment:
There is a common pattern where you have a conditional to handle situations when the value returned is None. Rather than calling the function twice in the case where it is not None, do:
    name = event.get_name_text()
    if name is not None:
        ret += ...
    else:
        ret += ...